test/integration/agent/openflow_test.go

```diff
@@ -80,7 +80,7 @@ func TestConnectivityFlows(t *testing.T) {
 	c = ofClient.NewClient(br)
 	err := ofTestUtils.PrepareOVSBridge(br)
 	if err != nil {
-		t.Errorf("failed to prepare OVS bridge: %v", br)
+		t.Errorf("failed to prepare OVS bridge: %v", err)
```

Comment:
```suggestion
		t.Errorf("Failed to prepare OVS bridge: %v", err)
```

Reply:
Updated according to the suggestion, thanks.